modules/ts/misc/run_suite.py

```diff
@@ -138,10 +144,25 @@ def runTest(self, path, logfile, workingDir, args = []):
             if isColorEnabled(args):
                 args.append("--gtest_color=yes")
             cmd = self.wrapInValgrind([exe] + args)
+            env = {}
+            if not self.options.valgrind and self.options.trace:
+                env['OPENCV_TRACE'] = '1'
+                env['OPENCV_TRACE_LOCATION'] = 'OpenCVTrace-{}'.format(self.getLogBaseName(exe))
```

Comment:
Could you please help me to locate where this env var is really used in the tracing code? Is it in a new code (cannot find it easily)? And when we set this var, do we automatically switch to a text backend instead of ITT backend. I was just trying to follow https://github.com/opencv/opencv/wiki/Profiling-OpenCV-Applications, and searching for this part "An alternative to the ITT-based instrumentation, is to trace the OpenCV calls in the compact machine-readable text form". Could you please point me to specific file/line when I can start reviewing this specific part?